src/autocomplete.js

```diff
@@ -274,16 +291,16 @@ class Autocomplete extends Component {
       () => {
         // call the provided callback if it's a callback
         cbToCb(cb)()
-        // if the selectedValue changed
```

Comment:
Or maybe we don't even worry about `prevState` since that should be fairly easy for someone to store 🤔